src/collectors/mesos/mesos.py

```diff
@@ -0,0 +1,68 @@
+# coding=utf-8
+
+"""
+
+Collects metrics from a mesos instance (master or slave).
+
+#### Example Configuration
+
+Typically metrics for the master are exposed on port 5050,
+while metrics for the slaves are exposed on port 5051.
+
+```
+    host = localhost
+    port = 5050
+```
+"""
+
+import diamond.collector
+import json
+import urllib2
+
+
+class MesosCollector(diamond.collector.Collector):
+
+    METRICS_PATH = "metrics/snapshot"
+
+    def get_default_config_help(self):
+        config_help = super(MesosCollector,
+                            self).get_default_config_help()
+        config_help.update({
+            'host': 'Hostname',
+            'port': 'Port',
+            'path': 'Metrics Path'
+        })
+        return config_help
+
+    def get_default_config(self):
+        config = super(MesosCollector, self).get_default_config()
+        config.update({
+            'host': 'localhost',
+            'port': 5050,
```

Comment:
Nah, I think that complicates too much. I wasn't sure if you could set different ports or not, but since you can, there's not much value in saying `is_master` or anything. If anything, I'd just comment that or put it in the port's help text. Could just be `Port (default 5050, but needs to be set to 5051 for mesos-slave)`